client/bulkwriter/bulk_import.go

```diff
@@ -0,0 +1,313 @@
+	// milvus params
+	URI            string     `json:"-"`
+	CollectionName string     `json:"collectionName"`
+	Files          [][]string `json:"files"`
+	// optional params
+	PartitionName string `json:"partitionName,omitempty"`
+	APIKey        string `json:"-"`
+	// cloud extra params
+	ObjectURL string `json:"objectUrl,omitempty"`
+	ClusterID string `json:"clusterId,omitempty"`
+	AccessKey string `json:"accessKey,omitempty"`
+	SecretKey string `json:"secretKey,omitempty"`
+
+	// reserved extra options
+	Options map[string]string `json:"options,omitempty"`
+}
+
+func (opt *BulkImportOption) GetRequest() ([]byte, error) {
+	return json.Marshal(opt)
+}
+
+func (opt *BulkImportOption) WithPartition(partitionName string) *BulkImportOption {
+	opt.PartitionName = partitionName
+	return opt
+}
+
+func (opt *BulkImportOption) WithAPIKey(key string) *BulkImportOption {
+	opt.APIKey = key
+	return opt
+}
+
+func (opt *BulkImportOption) WithOption(key, value string) *BulkImportOption {
+	if opt.Options == nil {
+		opt.Options = make(map[string]string)
+	}
+	opt.Options[key] = value
+	return opt
+}
+
+func NewBulkImportOption(uri string,
+	collectionName string,
+	files [][]string,
+) *BulkImportOption {
+	return &BulkImportOption{
+		URI:            uri,
+		CollectionName: collectionName,
+		Files:          files,
+	}
+}
+
+// NewCloudBulkImportOption returns import option for cloud import API.
+func NewCloudBulkImportOption(uri string,
+	collectionName string,
+	files [][]string,
+	apiKey string,
+	objectURL string,
+	clusterID string,
+	accessKey string,
+	secretKey string,
+) *BulkImportOption {
+	return &BulkImportOption{
+		URI:            uri,
+		CollectionName: collectionName,
+		Files:          files,
+		APIKey:         apiKey,
+		ObjectURL:      objectURL,
+		ClusterID:      clusterID,
+		AccessKey:      accessKey,
+		SecretKey:      secretKey,
+	}
+}
+
+type BulkImportResponse struct {
+	ResponseBase
+	Data struct {
+		JobID string `json:"jobId"`
+	} `json:"data"`
+}
+
+// BulkImport is the API wrapper for restful import API.
+func BulkImport(ctx context.Context, option *BulkImportOption) (*BulkImportResponse, error) {
+	url := option.URI + "/v2/vectordb/jobs/import/create"
+	bs, err := option.GetRequest()
+	if err != nil {
+		return nil, err
+	}
+
+	req, err := http.NewRequestWithContext(ctx, http.MethodPost, url, bytes.NewReader(bs))
+	if err != nil {
+		return nil, err
+	}
+	if option.APIKey != "" {
+		req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", option.APIKey))
+	}
+
+	result := &BulkImportResponse{}
+	err = doPostRequest(req, result)
+	if err != nil {
+		return nil, err
+	}
+	return result, result.CheckStatus()
+}
+
+type ListImportJobsOption struct {
+	URI            string `json:"-"`
+	CollectionName string `json:"collectionName"`
+	ClusterID      string `json:"clusterId,omitempty"`
+	APIKey         string `json:"-"`
+	PageSize       int    `json:"pageSize"`
+	CurrentPage    int    `json:"currentPage"`
```

Comment:
PageSize、CurrentPage milvus not support this two parameters, could be setted as omitempty